test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Client/ProxyExecutionManagerTests.cs

```diff
@@ -402,6 +402,17 @@ public void StartTestRunShouldCatchExceptionAndCallHandleRawMessageAndHandleLogM
             mockTestRunEventsHandler.Verify(s => s.HandleLogMessage(TestMessageLevel.Error, It.IsAny<string>()));
         }
 
+        [TestMethod]
+        public void StartTestRunForCancelRequestShouldHandleLogMessageWithProperErrorMessage()
```

Comment:
we can have a unit test for case when exception is not testplatformexception

Reply:
Added.